.github/workflows/test.yml

```diff
@@ -67,19 +67,24 @@ jobs:
     - name: Install krb5-user
       run: sudo apt-get install krb5-user
 
-    - name: Set up Go
-      uses: actions/setup-go@v3
+    - name: Setup Go
+      uses: actions/setup-go@v2
       with:
         go-version: '1.18'
-      id: go
+        check-latest: true
 
-    - name: Check out code into the Go module directory
+    - name: Check out code
       uses: actions/checkout@v3
 
-    - name: TF acceptance tests
+    - name: Setup Terraform ${{ matrix.terraform }}
+      uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: ${{ matrix.terraform }}
+        terraform_wrapper: false
+
+    - name: Run acceptance test
       timeout-minutes: 30
```

Comment:
Although maybe we could drop this other timeout, since the top level one probably covers it well enough. 😄 

```suggestion
```